RELEASE.md

```diff
@@ -1,7 +1,12 @@
 # Release Process for OLM v1
 
 ## Choosing version increment
-When making releases for operator-controller, version increments should adhere strictly to Semantic Versioning. In short:
+
+The `operator-controller` project is in its initial development phase and has yet to have a Major release, so users should assume that breaking changes may be seen in Minor releases as defined [here](https://semver.org/#spec-item-4).
+
+In general, the `operator-controller`` will only support Minor releases until it has reached a degree of stability and adoption that the benefits of supporting Patch releases outweighs the costs of supporting this release workflow. If a member of the community strongly desires a patch release addressing a critical bug, they should submit an issue and it will be considered on a case-by-case basis. 
+
+In the future, we hope to move to a model in which version increments adhere strictly to Semantic Versioning. In short:
```

Comment:
Technically, we _are_ already adhering to semver versioning because the semver spec has a carve out for breaking changes being permissible in `0.Y` versions.

Reply:
Perhaps something like 
```suggestion
In the future, the `operator-controller` will have a major release in which we'll adopt the following Semantic Versioning guarantees:
```

Reply:
Your edit lgtm.